index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+    <head>
+      <link rel="stylesheet" href="style.css">
+
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+        <header>
+        <section class="toolbar">
+            <div class="tbar">
+                <div class="menu-bar">
+                    <p class="john-doe">IKOOTE</p>
+          
+            <img src="./images/Icon - Menu.png" class="icon-Menu">
+                </div>
+            </div>
+
+        </section>
+        <section class="headline">
+            <div> 
+                <h1>Hey There. I’m Ikoote </h1>
+            <h2>I’am a Software Developer</h2>
+            <p class="txt">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+            </div>
+            <p style="background-image: url('./images/Illustration.svg');" class="illust"> 
+           
```

Comment:
- [ ] Great work making the changes requested by the previous reviewer. However, it is not a good HTML/CSS practice to use inline styles when making CSS declarations. I would advise you to put this declaration in your style sheet. For more information, you can read this [article](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)